packages/ecs-browser/src/ComponentEditor.tsx

```diff
@@ -24,11 +24,17 @@ export const ComponentEditor = ({
     <ComponentEditorContainer>
       <ComponentTitle>
         {component.id}
-        <ComponentBrowserButton onClick={() => {
-          removeComponent(component, entity);
+        {setContractComponentValue && (
+          <ComponentBrowserButton
+            onClick={() => {
+              removeComponent(component, entity);
```

Comment:
Removing local components should be possible even without `setContractComponentValue` function
```suggestion
          <ComponentBrowserButton
            onClick={() => {
              removeComponent(component, entity);
```